core/src/main/java/org/apache/gravitino/meta/TableEntity.java

```diff
@@ -36,6 +37,8 @@ public class TableEntity implements Entity, Auditable, HasIdentifier {
   public static final Field NAME = Field.required("name", String.class, "The table's name");
   public static final Field AUDIT_INFO =
       Field.required("audit_info", AuditInfo.class, "The audit details of the table");
+  public static final Field COLUMNS =
+      Field.optional("columns", ColumnEntity[].class, "The columns of the table");
```

Comment:
Why do you mark it as `optional`? Just to make it compatible with the previous version?

Reply:
Yes, it is mainly for compatible thinking.